go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -169,6 +169,24 @@ func (sparkResourceHandler) BuildResource(ctx context.Context, taskCtx pluginsCo
 	}
 	sparkConfig["spark.kubernetes.executor.podNamePrefix"] = taskCtx.TaskExecutionMetadata().GetTaskExecutionID().GetGeneratedName()
 
+	// Add driver/executor defaults to CRD Driver/Executor Spec as well.
+	cores, err := strconv.Atoi(sparkConfig["spark.driver.cores"])
```

Comment:
so there is no way to run flyte with older spark operator right? otherwise this will fail

Reply:
also will this make it possible to run Spark now in client mode?

Reply:
The latest plugin version creates v1beta2 so will need this fix to work correctly. The older versions should continue to work.

Client mode  will need a more work to run the submit-job as a pod and set its resources correctly i.e. driver memory + cores .